src/show/styles/style.css

```diff
@@ -3,4 +3,6 @@
 }
 .texthead {
     font-size: 1.3rem;
-}
\ No newline at end of file
+}
```

Comment:
why we have styles here, move it next to the folder we have or if this is used more often then create a css in tailwind.config.json

Reply:
sir i didn't get this (move it next to the folder we have ) in which folder i have to move.